7_react/react1/week2/react-app/src/components/List.js

```diff
@@ -0,0 +1,88 @@
+import React, { useEffect, useState } from "react";
+import { ListItem } from "./ListItem";
+
+export function List() {
+  const [list, setList] = useState(initialList);
+
+  useEffect(() => {
+    console.log("something");
+  }, [list]); // dependency on list changes
+
+  useEffect(() => {
+    console.log("hello");
+  }, []); // empty dependency array = only when the component mounts;
```

Comment:
I assume you added this code to help yourself debug code and see things running. Since you are not doing anything besides writing a console.log consider to remove this redundant code before raising your PR. 

In the real world it is very common that we add temporary code while working on a task but we should strive to clean it up before submitting the final PR. Like when you are cooking it is perfectly fine to be a bit messy but ideally you should clean up the kitchen before you serve the food so that it is neat and ready to be used next time.